test/unit/api/core/io/json/grammar_test.rb

```diff
@@ -40,10 +40,10 @@ class Core::Io::Json::GrammarTest < ActiveSupport::TestCase
       stream.expects(:block).with(:attribute_name).yields(nested_stream)
 
       children =
-        ['Child 1', 'Child 2'].map do |name|
+        ['Child 1', 'Child 2'].to_h do |name|
```

Comment:
Bit surprised Rubocop doesn't suggest index_with in a few of these cases. I've seen it suggest it as a replacement for each_with_object in a few cases. Great method, horrible name.